src/include/sof/audio/drc/drc.h

```diff
@@ -9,6 +9,7 @@
 
 #include <stdint.h>
 #include <sof/audio/drc/drc_plat_conf.h>
+#include <sof/audio/buffer.h>
```

Comment:
https://github.com/thesofproject/sof/blob/main/src/audio/drc/drc.c#L9C30-L9C30

drc.c already include buffer.h, what's the purpose to add here? if needed add here, should above be deleted?

Reply:
Line 61 doesn't compile without this

Reply:
Removed from the c file

Reply:
const struct audio_stream __sparse_cache *source, ? this line, if this is the case, maybe simply add:
struct audio_stream;
before line 60 is fine.

Reply:
I would rather just include the proper header unless you have a reason not to

Reply:
here actually just forward declarations *might* be enough

Reply:
I think including buffer.h here is ok. Forward declarations make sense in interface headers that are included in a large number of files, but I don't think we need to go overboard with this in files like drc.h that are only included in a handful of places.